jobs/BUILD

```diff
@@ -5,6 +5,7 @@ py_binary(
     srcs = ["config_sort.py"],
     data = [
         ":jobs",
+        "//boskos:resources.json",
```

Comment:
eh, default visibility for boskos is public, assume I don't need to export explicitly?